Source/Glottis.cpp

```diff
@@ -102,17 +97,10 @@ void Glottis::finishBlock()
 		vibrato += 0.2 * simplex1(this->totalTime * 0.98);
 		vibrato += 0.4 * simplex1(this->totalTime * 0.5);
 	}
-	if (this->UIFrequency > this->smoothFrequency)
```

Comment:
Maybe refactor this to a method, but just don't call it. Would be good to keep this around, we could use it to make the voice sound more natural.